### ISSUE TITLE
New submission image style

### ISSUE THREAD

**Issue by elenmerbau** [Tunous/Dawn#186]
First of all, feels good to see a new release, good job!

My first problem is within the new viewing style - large previews. They are not large enough, not a full preview to be worth eating space on the screen, and yet they are too small to be able to show everything for you to just scroll without viewing the post itself. They are somewhere in the middle, partially serving both purposes at the same time.

I think, the better solution is to either keep it compact with "Thumbnail" default style or make an option for a full sized preview instead.

Second problem is the separation line between posts in "Large" preview style, it's too big.
![image](https://user-images.githubusercontent.com/23418072/80913470-7cba6080-8d0a-11ea-9ba2-093fb5a854b8.png)

**Comment by Tunous** [comment]
Both changes are included in #207

@elenmerbau 
Would you mind checking debug version of that pull request and giving feedback what do you think about it?

You can find mentioned version here: https://github.com/Tunous/Dawn/suites/675828218/artifacts/6160923

**Comment by elenmerbau** [comment]
It looks good! The only thing bothering me is that there's no clear separation between posts and you can get lost pretty easily, not sure what's the optimal solution for it yet. Maybe make lines between posts colored like comments are or something.

**Comment by Tunous** [comment]
Maybe someone knows another app with similar layout where they solved such problem?

**Comment by elenmerbau** [comment]
Here are 2 examples from Relay and Sync. 
![relay](https://user-images.githubusercontent.com/23418072/81917458-d4728a80-959a-11ea-9ac2-ac018d8171fd.png) ![sync](https://user-images.githubusercontent.com/23418072/81917462-d5a3b780-959a-11ea-96a5-0f3f192bf2c0.png)

And this how it looks like in Dawn, ~4 px separator:
![Screenshot_20200514-152728](https://user-images.githubusercontent.com/23418072/81920077-4bf5e900-959e-11ea-9bdd-995bb0a075d3.png)

**Comment by elenmerbau** [comment]
Can go even further - app's accent color colored separators.
![Untitled](https://user-images.githubusercontent.com/23418072/82077692-75e60300-96a5-11ea-8d93-e4e8295debfb.png)

**Comment by Donkey-Doug** [comment]
I agree with the divider to seperate posts. The accent color is a nice touch, but a simple black line for a light themed app or a white line for a dark theme app is also functional.
